src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -308,6 +292,9 @@ public void runDevAppServer1Command(List<String> jvmArgs, List<String> args,
     command.addAll(args);
 
     logCommand(command);
+
+    Map<String, String> environment = Maps.newHashMap();
+    environment.put("JAVA_HOME", javaBinaryPath.getParent().getParent().toString());
```

Comment:
You mean devappserver1 doesn't internally use JAVA_HOME? I was under the impression it did, after talking with @loosebazooka and @ludoch ?